packages/kbn-performance-testing-dataset-extractor/src/extractor.ts

```diff
@@ -75,15 +82,16 @@ const getTraceItems = (
       return {
         processor: hit.processor,
         traceId: hit.trace.id,
-        timestamp: hit['@timestamp'],
+        date: hit['@timestamp'],
```

Comment:
Maybe it is redundant, but I decided to store both Date and timestamp number: the first one is mainly for debug purpose in case of trouble and the other one is used by Gatling simulation.